manager/src/main/resources/service_limits.json

```diff
@@ -0,0 +1,23 @@
+{
+  "instance_limits": [{
+    "instance_type": "EVAL",
+    "processor_limit": "6",
+    "bridge_duration": "PT12H"
+  },
+    {
+      "instance_type": "STANDARD",
+      "processor_limit": "30"
+    }
+  ],
+  "default_quotas": [{
+    "instance_type": "EVAL",
+    "quota": "1"
+  }],
+  "organisation_quotas": [{
```

Comment:
@vaibhavjainwiz I'd suggest we remove this override and just allow the default to be `eval` instances which allow `30` processors. This will be more than enough to support our local dev and CI pipelines.

For our `dev` and `stage` environments, we can create a `ConfigMap` with the `service_limits.json` in it and mount it on the Fleet Manager pod to set some reasonable limits in those environments.

Reply:
How many EVAL instance should we allowed to be created in default?
Below is the service-limit.json syntax
```
{
  "quota_limits": [{
    "quota_type": "EVAL",
    "processor_limit": "30",
    "bridge_duration": "PT12H"
  },
    {
      "quota_type": "STANDARD",
      "processor_limit": "30"
    }
  ],
  "default_quotas": [{
    "quota_type": "EVAL",
    "quota": "1"
  }]
}
```

Reply:
setting it to 30